lib/exec.js

```diff
@@ -37,14 +47,15 @@ async function exec (cmd, args = [], opts = {}) {
     let stdoutArr = [], stderrArr = [], timer = null;
 
     // if the process errors out, reject the promise
-    proc.on('error', (err) => {
+    proc.on('error', /** @param {NodeJS.ErrnoException} err */(err) => {
+      // @ts-ignore
       if (err.errno === 'ENOENT') {
```

Comment:
i mean if you want to update this and there's a test for it, go for it!

Reply:
i'll do it in another PR, maybe